### PR TITLE
add comments from Lars

### PR DESCRIPTION
**Pull request by fieker**

### CODE REVIEW

**Comment by thofma** [comment]
Interesting. What was the result for your example? Who is the winner?

**Review by thofma — APPROVED** [review]

File: src/NumberTheory/NmbThy.jl [hunk 1]
```diff
@@ -154,6 +153,37 @@ function irreducibles(S::Vector{NfAbsOrdIdl{AnticNumberField,nf_elem}})
   return res
 end
 
+#= From Lars:
```
Comment (fingolfin) [inline src/NumberTheory/NmbThy.jl:156]: @fieker I am confused by this PR. What is this new comment about?!

Comment (thofma) [reply src/NumberTheory/NmbThy.jl:156]: From the context it seems to show how to use different backends for the `solve_non_negative` method.

Comment (fieker) [reply src/NumberTheory/NmbThy.jl:156]: Indeed, It did not solve the original problem, but I thought this helpful enough so it should be preserved in a useful spot.
The original provlem involved a largish solve problem that allowed for "trivial" preprocessing, there were quite some inqualtities if the type -2*x_i > 0 or 3x <= 2 which in this context can easily be removed (the entire variable).
This was an attempt, by Lars, to try the different backends. The result was not quite as predicted by Michael...

**Pull request merged**